jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -209,13 +209,17 @@ def installationGitUrl = params.installationGitUrl ?: 'git@github.com:integr8ly/
 def installationGitRef = params.installationGitRef ?: 'master'
 def githubToken = params.githubToken ?: 'jenkins-github-api-token'
 def githubCredentialsID = params.credentialId ?: 'jenkinsgithub'
-def manifestVar = params.manifestVar
+def releaseTagVar = params.manifestVar ?: params.releaseTagVar
+def productVersionVar = params.productVersionVar
 def projectOrg = params.projectOrg
 def projectRepo = params.projectRepo
 def productName = params.productName
 def releaseFetchMethod = params.releaseFetchMethod
+def templateLocation = params.productTemplateLocation
```

Comment:
Is this used?

Reply:
That should've been called `productVersionLocation`